.github/workflows/buildRelease.yml

```diff
@@ -86,6 +86,76 @@ jobs:
         name: wheels
         path: dist
 
+  build_linux_arm64:
+    name: Manylinux arm64 Build
+    runs-on: linux-arm64
+    strategy:
+      matrix:
+        package: [ torch-mlir, torch-mlir-core ]
+        py_version: [ cp311-cp311 ]
+
+    steps:
+
+    - name: Prepare workspace
+      run: |
+        # Clear the workspace directory so that we don't run into errors about
+        # existing lock files.
+        sudo rm -rf $GITHUB_WORKSPACE/*
+
+    - name: Get torch-mlir
+      uses: actions/checkout@v3
+      with:
+        submodules: 'true'
+        fetch-depth: 0
+
+    - uses: ./.github/actions/setup-build
+      with:
+        cache-suffix: 'release'
```

Comment:
We should use a different suffix so that the key for cache artifacts does not collide between x64 and arm64 builds.  Tangentially related, do we want to disable caching for release builds?

Reply:
Good catch. We should just disable for release builds

Reply:
Addressed in https://github.com/llvm/torch-mlir/pull/2168, although it looks like this PR might need to be rebased.